<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History</class>
 <widget class="QWidget" name="History">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>历史记录</string>
  </property>
  <widget class="QComboBox" name="HistoryList">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>341</width>
     <height>31</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>选择历史对局打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QLabel" name="board_preview">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>241</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>681</width>
     <height>511</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/pictures/board_background.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="save_delete">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>HistoryList</zorder>
  <zorder>pushButton</zorder>
  <zorder>board_preview</zorder>
  <zorder>save_delete</zorder>
  <zorder>pushButton_2</zorder>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
